README.md

```diff
@@ -31,36 +31,144 @@ const MyComponent = () => {
 ## ✨ Features
 
 * ES6 Fetch powered by React Hooks
-* Centralized Cache
-* Redux, Headless (Coming Soon) or `useReducer` (Coming Soon) Cache modes available
+* Pure React or Redux Shared Cache modes available
+* Headless per-Hook Cache Mode available
 * SSR-friendly
 
 ## ⬇️ Install
```

Comment:
Should this be `🤹‍ Usage` instead? This section goes over what to install and how to use it

Reply:
No. This is how to install these. Usage is farther down in another section.

Reply:
I will settle for `Install & Setup` or `Install and Setup` whichever makes a nicer link

Reply:
https://github.com/americanexpress/fetchye/blob/docs/readme/README.md#%EF%B8%8F-install--setup

Reply:
Fixed in https://github.com/americanexpress/fetchye/pull/12/commits/7c5e59763561b539fa6c985bcad2e78ce539435e